src/kinematics/SwerveDriveKinematics.h

```diff
@@ -0,0 +1,113 @@
+#pragma once
+
+#include "../navtypes.h"
+
+using namespace navtypes;
+
+struct swervewheelvel_t {
+	/**
+	 * Represents robot wheel velocities in robot reference frame,
+	 * in polar form (wheel speed and angle)
+	 */
```

Comment:
Can you put this comment before the struct (not inside it) so it renders in doxygen correctly?